app/Server.php

```diff
@@ -30,7 +55,7 @@ public function bbb()
      */
     public function getMeetings()
     {
-        if (!$this->status or $this->offline) {
+        if (!$this->status) {
```

Comment:
Why when the server is offline the meetings should be returned?

Reply:
Status is manually set (enable/disable) server, whereas offline is just an indication if the server is currently offline. To detect that the server is online again, the same function is called so it needs to respond with fresh data.